qiskit/pulse/reschedule.py

```diff
@@ -171,19 +174,20 @@ def add_implicit_acquires(schedule: ScheduleComponent, meas_map: List[List[int]]
     return new_schedule
 
 
-def pad(schedule: Schedule, channels: Optional[Iterable[Channel]] = None,
+def pad(schedule: Schedule,
+        channels: Optional[Iterable[Channel]] = None,
         until: Optional[int] = None) -> Schedule:
-    """Pad the input Schedule with `Delay`s on all unoccupied timeslots until
-    `schedule.duration` or `until` if not `None`.
+    """Pad the input ``Schedule`` with ``Delay``s on all unoccupied timeslots until ``until``
```

Comment:
The apidoc generated for this looks like
<img width="965" alt="image" src="https://user-images.githubusercontent.com/11485594/74470281-3de3df80-4e6c-11ea-9123-181d10873390.png">


```suggestion
    """Pad the input ``Schedule`` with ``Delay``\'s on all unoccupied timeslots until ``until``
```

The suggestion will generate 
<img width="929" alt="image" src="https://user-images.githubusercontent.com/11485594/74470344-5d7b0800-4e6c-11ea-8c57-8fb7d7ddbc21.png">

Reply:
I will make this change in a follow up PR

Reply:
Thank you. 😊